schemas/outcomes-metrics.v1.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "metrics_outcome",
+  "$ref": "#/definitions/MetricsOutcome",
+  "definitions": {
+    "MetricsOutcome": {
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "description": "The UNIX timestamp in seconds when the outcome was created, optionally truncated to the last hour.",
+          "$ref": "#/definitions/U64"
+        },
+        "org_id": {
+          "description": "The organization for which this outcome is being sent.",
+          "$ref": "#/definitions/U64"
+        },
+        "project_id": {
+          "description": "The project for which this outcome is being sent.",
+          "$ref": "#/definitions/U64"
+        },
+        "mri": {
+          "description": "Valid metric resource identifier `<type>:<namespace>/<name>[@<unit>]` for which the outcome is being sent.",
+          "type": "string"
+        },
+        "outcome": {
+          "description": "Outcome ID, metric outcomes share the same numeric outcome ID with regular outcomes.",
+          "$ref": "#/definitions/U64"
+        },
+        "reason": {
+          "description": "Optional machine readable, free-form reason code.",
+          "type": ["string", "null"]
+        },
+        "quantity": {
+          "description": "Amount of metric buckets accepted by Relay (volume).",
+          "$ref": "#/definitions/U64"
+        },
```

Comment:
Same as below. What's the time range we are talking about here? Is it fixed or is it dependent on relay bucketing time ?

Reply:
This is independent of time, this is the amount of statsd elements Relay receives and parses out of envelopes without any aggregation.